content/develop/interact/search-and-query/administration/configuration.md

```diff
@@ -0,0 +1,582 @@
+---
+aliases: /develop/interact/search-and-query/basic-constructs/configuration-parameters
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: 'Redis Query Engine can be tuned through multiple
+  configuration parameters. Some of these parameters can only be set at load-time,
+  while other parameters can be set either at load-time or at run-time.'
+linkTitle: Configuration parameters
+title: Configuration parameters
+weight: 1
+---
+{{< note >}}
+As of Redis Community Edition 8.0, the `FT.CONFIG SET` and `FT.CONFIG GET` commands are deprecated.
+Configuration parameters for Redis Query Engine are now set in the following ways:
+* At load-time via your `redis.conf` file.
+* At run-time (where applicable) using the [`CONFIG SET`]({{< relref "/commands/config-set" >}}) command.
+{{< /note >}}
+
+## Redis Query Engine configuration parameters
+
+The following table summarizes which configuration parameters can be set at run-time, and compatibility with Redis Enterprise and Redis Cloud.
+
+| Old parameter name                | New parameter name                                              | Run-time             | Redis<br />Enterprise | Redis<br />Cloud |
+| :-------                          | :-----                                                          | :-----------         | :--| :-- |
+| BG_INDEX_SLEEP_GAP                | [search-bg-index-sleep-gap](#search-bg-index-sleep-gap)         | :white_large_square: |||
```

Comment:
What does the white square mean, as opposed to leaving the cell blank?

Reply:
It means not available. This is the way it was done on the previous versions of these pages.